tests/unit/lms/services/application_instance_service_test.py

```diff
@@ -37,11 +43,11 @@ def service(self, db_session, pyramid_request):
         return ApplicationInstanceService(db=db_session, request=pyramid_request)
 
     @pytest.fixture(autouse=True)
-    def application_instance(self):
+    def application_instance(self, application_instance):
         # Some noise
         factories.ApplicationInstance.create_batch(size=3)
 
-        return factories.ApplicationInstance()
+        return application_instance
```

Comment:
It's possible to use the main fixture now.

Reply:
Does this fixture have anything to do with the `application_instance` anymore?

Could this be:

```python
@pytest.fixture(autouse=True)
def with_application_instance_noise(self):
    factories.ApplicationInstance.create_batch(size=3)
```